lib/Traits/TraitsClasses/CartoMapV3CatalogItemTraits.ts

```diff
@@ -0,0 +1,62 @@
+import primitiveArrayTrait from "../Decorators/primitiveArrayTrait";
+import primitiveTrait from "../Decorators/primitiveTrait";
+import mixTraits from "../mixTraits";
+import { GeoJsonTraits } from "./GeoJsonTraits";
+
+export default class CartoMapV3TableCatalogItemTraits extends mixTraits(
+  GeoJsonTraits
+) {
+  @primitiveTrait({
+    type: "string",
+    name: "Access token",
+    description: "The access token to pass to the Carto Maps API"
+  })
+  accessToken?: string;
+
+  @primitiveTrait({
+    type: "string",
+    name: "Authorization token",
+    description: "The authorization token to pass to the Carto Maps API"
+  })
+  connectionName = "carto_dw";
+
+  @primitiveTrait({
+    type: "string",
+    name: "Base URL",
+    description:
+      'Base URL for Carto API (eg "https://gcp-us-east1.api.carto.com/")'
+  })
+  baseUrl = "https://gcp-us-east1.api.carto.com/";
```

Comment:
I suppose this is the default if a baseUrl trait is not provided in a catalog item definition? So is `gcp-us-east1` the best or maybe `gcp-australia-southeast1.api.carto.com` like in the examples? More of our users in Aus or not?

Reply:
Good idea, I'll change it